link_shortener/tests/test_form_views.py

```diff
@@ -0,0 +1,55 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+from unittest import TestCase
+
+from link_shortener.server import create_app
+
+
+class TestFormViews(TestCase):
```

Comment:
Is it that you like `unittest` more than `pytest` or was thare a problem with `pytest` that you work around using this? I mean, people generally don't like `unittest` and prefer to use the `pytest` style so if there was a problem, I can help you to write it the pytest way...

Reply:
It's the Django way. 😄  
That being said, I can't use Django's APIClient anyway, so I can change it to pytest if you'd like.

Reply:
Well, actually I like `unittest` (well, just because it's part of std lib; in fact, I hate the interface) so I am fine with this. But during my 15 year old career, I've never met anyone who writes tests this way. Every team I've ever been part of used the pytest style. So, I am happy to leave this up to you. If you want to practice the the (apparently) preferred style of writing pytest fixtures, feel free to do so... :)

Reply:
I'll keep unittest, then 😄 I prefer its syntax.